uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/BaseSettings.java

```diff
@@ -81,4 +84,17 @@ public void launchScreenLockPage() {
         Logger.i(TAG, "Launching Screen Lock Page..");
         launchIntent(Settings.ACTION_SECURITY_SETTINGS);
     }
+
+    @Override
+    public void launchAppInfoPage(@NonNull final String packageName) {
+        try {
+            //Open the specific App Info page:
+            final Uri data = Uri.parse("package:" + packageName);
+            launchIntent(Settings.ACTION_APPLICATION_DETAILS_SETTINGS, data);
+        } catch (final ActivityNotFoundException e) {
+            Logger.e(TAG, "Package: " + packageName + " probably not available on device.", e);
+            //Open the generic Apps page:
+            launchIntent(Settings.ACTION_MANAGE_APPLICATIONS_SETTINGS);
```

Comment:
we should just fail here

Reply:
we can't do the test if Chrome isn't there? I guess we could install it from PlayStore....but for now let's just fail it

Reply:
Addressed